ResearchKit/Common/ORKFormItemCell.h

```diff
@@ -63,6 +63,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, copy, nullable) id defaultAnswer;
 @property (nonatomic) CGFloat maxLabelWidth;
 @property (nonatomic) CGFloat expectedLayoutWidth;
+@property (nonatomic) NSMutableDictionary   * __nullable __strong * __nullable savedAnswers;
```

Comment:
Is `**` required? 
And if the cell is not going to modified the dictionary, then just declare  `NSDictionary`, which is more safe.